alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -717,17 +738,36 @@ object DocumentLevelMonitorRunner : MonitorRunner() {
      */
     private fun transformDocumentFieldNames(
```

Comment:
this is a very critical piece fo code. can we update java docs description of this method incorporating the new change?

Reply:
updated the documentation for the method.